sql/moz-fx-data-shared-prod/account_ecosystem_derived/fxa_logging_users_daily_v1/metadata.yaml

```
@@ -12,5 +14,5 @@ scheduling:
   # We access a restricted table for getting an HMAC key, so cannot dry run
   # and must explicitly list referenced tables.
   referenced_tables:
-    - ['moz-fx-data-shared-prod', 'firefox_accounts_stable', 'account_ecosystem_v1']
-    - ['moz-fx-data-shared-prod', 'account_ecosystem_derived', 'ecosystem_user_id_lookup_v1']
+    - ["moz-fx-data-shared-prod","firefox_accounts_stable","account_ecosystem_v1",]
+    - ["moz-fx-data-shared-prod","account_ecosystem_derived",ecosystem_user_id_lookup_v1',]
```

This is missing proper quotes

```suggestion
    - ["moz-fx-data-shared-prod","account_ecosystem_derived", "ecosystem_user_id_lookup_v1",]
```